debug_src/debug.js

```diff
@@ -0,0 +1,78 @@
+const preact = require('../');
+
+const original = {
+	h: preact.h
+};
+
+const config = {
+	undefinedComponents: true,
+	stringRefs: true,
+	arrayChildren: true
+};
+
+preact.h = function(nodeName, attributes) {
```

Comment:
It could, but that way you won't get good stack trace, i.e. the call before throwing the error will point directly to the place of `h()` invocation. If this won't provide good stack traces I'm not really interested in having this package myself.